### PR TITLE
Bookstore: Use Redux in React components

### PR DESCRIPTION
**Pull request by jugosack**
For this pull request, it is completed the following [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/use_redux_in_react_components_v2.md):

- Connect the store to the main component
- Define a state slice that will contain your books
- Display your books, received from the slice, in a reusable component
- Dispatch actions using useDispatch
- Add a Button component, which includes an event handler that adds a book to the books array (with attributes id, title, and author)
- Add a Button component, which includes an event handler that removes a book from the books array (by id)
- Add a Button component, which includes an event handler that checks status (i.e add functionality for "Check status" button click)

### CODE REVIEW

File: src/components/addBook.jsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const AddBook = () => (
+  <button type="submit">Add book</button>
```
Comment (FrederickMih) [inline src/components/addBook.jsx]: - [ ] Based on the requirements of this project, this button should include an event handler to add a book to the `books` array(with attributes title and author). Let's kindly adhere to the requirement of this project.

File: src/components/removeButton.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types';
+
+const RemoveButton = (props) => {
+  const { onClick } = props;
+  return (
+    <button type="button" onClick={onClick}>
+      Remove
+    </button>
```
Comment (FrederickMih) [inline src/components/removeButton.jsx]: - [ ] You have added the event handler but the removeBook action(with the id attribute) is not being dispatched to the redux store as requested.


 <Button onClick={() => dispatch(removeBook(id))}>Remove</Button>

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand:  @jugosack, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!





## Highlights :green_circle: 

- PR is professional :+1: 
- Adding and removing books was still possible :+1: 
- Linter checks passed :heavy_check_mark: 


## Required Changes ♻️

- [ ] @jugosack, currently the MIT license link at the end of the readme file isn't working. Let's kindly fix the link in order to render the readme file professional :+1: 


_Check the comments under the review._

## Optional suggestions
**N/A**
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @jugosack  👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉

## Highlights
✔ Descriptive PR 👍 

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**